src/node/run-style-dictionary.js

```diff
@@ -30,45 +32,96 @@ async function cleanPlatformOutputDirs() {
 }
 
 export async function rerunStyleDictionaryIfSourceChanged(file) {
-  const { source } = styleDictionaryInstance.options;
```

Comment:
This no longer works because source is only the token source files.

Now, we can have source files that aren't tokens themselves, but may be imported into token files.

So instead, we will go through the platforms buildPaths and make a collection of all of the "output files". 
Then we filter "all files" by "output files" and we're left with all source files :)